README.md

```diff
@@ -111,10 +111,97 @@ Here is the summary of existing routes at the moment:
   * Post: `/news/my-post`
   * Anything else: `/foo/bar`
 
+## Extensions
+The platform can be extended in the following ways:
+
+### Theme Routes
+You can add a `routes.js` file to your theme. Define a module.export with a function that receives the express `app` object, and use it to add routes, middleware, etc, to the frontend app.
+
+*NOTE* You do not need to return anything from this function, just extend the app object by reference.
+
+Add the following to your `.env` file (or to your node environment via any available  method):
+`CKAN_THEME_ROUTES: "your_cool_theme"`
```

Comment:
@starsinmypockets isn't it redundant if we have `THEME` env var already that tells the app where to look for your theme stuff? Can't we use that?